project/.pre-commit-config.yaml

```diff
@@ -1,47 +1,32 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
+# see https://github.com/VOLTTRON/volttron-core/
 repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.0.1
+  rev: v4.1.0
   hooks:
-  - id: check-added-large-files
-  - id: check-toml
   - id: check-yaml
+  - id: check-json
+  - id: check-toml
+  - id: check-xml
+  - id: forbid-new-submodules
   - id: end-of-file-fixer
   - id: trailing-whitespace
-- repo: https://github.com/psf/black
-  rev: 22.3.0
-  hooks:
-  - id: black
-- repo: https://github.com/pre-commit/mirrors-mypy
-  rev: v0.910-1
-  hooks:
-  - id: mypy
-    exclude: ^(docs/|example-plugin/|tests/fixtures)
-- repo: local
-  hooks:
-  - id: pytest
```

Comment:
Let's also keep this hook because this precommit file will be used by the CI workflow to run tests. See https://github.com/VOLTTRON/copier-poetry-volttron-agent/blob/develop/.github/workflows/ci.yml#L73